dev/main.sh

```diff
@@ -47,8 +47,8 @@ if [ "$existingTenants" == "[]" ]; then
     for tenant in "${tenants[@]}"
     do
         echo "🐈Provisioning tenant: $tenant"
-        baseURL="$tenant.stellar.local:8000"
-        sdpUIBaseURL="$tenant.stellar.local:3000"
+        baseURL="http://$tenant.stellar.local:8000"
+        sdpUIBaseURL="http://$tenant.stellar.local:3000"
```

Comment:
Do we actually have https in place for this? I think it'd be `http`, right?

Reply:
Sorry, I don't get it.